app/helpers/concerns/with_student_path_navigation/revisit_navigation.rb

```diff
@@ -1,5 +1,15 @@
 module WithStudentPathNavigation
   class RevisitNavigation < Navigation
+
+    def button(navigable)
+      sibling = sibling_for(navigable)
+      if sibling && sibling != navigable
+        link_to link_icon(sibling), sibling, merge_confirmation_classes(navigable, class: clazz)
```

Comment:
Some thoughts: 

* why can't we move this logic to the top `Navigation` hierarchy? 
* we are repeating logic
* we are not solving the problem when we are on path's end - ok, I understand we can "solve" this by prohibiting reading in last position :thinking: . 
* also, if we apply this change, we are going to redirect user to home page when revisiting the exercise, instead of redirecting to the next chapter :unamused: : 

https://github.com/mumuki/mumuki-laboratory/blob/ac23edd5ae53fb7a21cbac27340f98491fdbd9ad/app/helpers/concerns/with_student_path_navigation.rb#L8

Reply:
To sum up, correct me if I am wrong but we are fixing a nasty bug with.... an also nasty behavior in a nasty way. 

I propose two alternatives  

* accept this new behavior as "bad, but still better than what we have", but refactor this code a little, so that logic is not repeated - we could use a template method in `Navigation`
* do it right - refactor this awful navigation hierarchy - so that it always points to the next chapter - which is currently not working, either :disappointed: - and this button always has the confirmation class, and we display the go to home button only when there is actually nothing else more to do here.

Reply:
@julian-berbel please share your thoughts here